### DISCUSSION TITLE
Help wanted Y-axes is not working

### DISCUSSION THREAD

**Discussion by Bjorn56** [arkypita/LaserGRBL#1124] [Q&A]
I have a new laser and I intended to use it for engraving.
It is a laser Diy Engraver 
 It looks like one of the axis are not working.  I tried to engrave a square and here is the result
![image](https://user-images.githubusercontent.com/71401014/93370336-f33aa380-f850-11ea-9ee9-33a7ccd1ceb3.png)
!

**Comment by Bjorn56** [comment]
it should be like this
![1024px-Kvadrato svg](https://user-images.githubusercontent.com/71401014/93370573-4f9dc300-f851-11ea-8f2d-06c972a1599f.png)

**Comment by arkypita** [comment]
1) Check electric cables
2) check belt tension (not too tight, not to soft)
3) check motor current setpoint, stepper driver usually has a trimmer to set motor current)
4) maybe a died driver?
5) check grbl config (resolution, speed, acceleration)

Do some specific test to understand what kind of problem. Try run motors without load, does they move or not? Try swap x/y cable, the problem is on y again or switch to x?

**Comment by StuartB4** [comment]
> GRBL configuration?

Start LaserGRBL, click Connect, then click Grbl menu and click Grbl configuration. Take a screenshot or clear photo of the settings list and post it here.

**Comment by Bjorn56** [comment]
IT sere the wheels at the laser. I have loodsened them and nog it is working. The only thing is there seems to be a lack of power becuase i can hardly see ten engravement

Verstuurd vanaf mijn iPad

Op 16 sep. 2020 om 22:45 heeft Stuart <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


GRBL configuration?

Start LaserGRBL, click Connect, then click Grbl menu and click Grbl configuration. Take a screenshot or photo of the settings list and post it here.

**Comment by Bjorn56** [comment]
Hi thanks for your advice.
I loosen the boults and now it is working ok. I have another question. The laser seems lacking power. It can't  engrave (blanc) wite wood. how can I adjust the laser so the focus point is just on the surface of the object?
 I like to hear form you
 Warm Regards,
Bjorn Vosters

________________________________
Van: arkypita <notifications@github.com>
Verzonden: woensdag 16 september 2020 19:42
Aan: arkypita/LaserGRBL <LaserGRBL@noreply.github.com>
CC: Bjorn56 <bjorn_vosters@hotmail.com>; Author <author@noreply.github.com>
Onderwerp: Re: [arkypita/LaserGRBL] Help wanted Y-axes is not working (#1124)


  1.  Check electric cables
  2.  check belt tension (not too tight, not to soft)
  3.  check motor current setpoint, stepper driver usually has a trimmer to set motor current)
  4.  maybe a died driver?
  5.  check grbl config (resolution, speed, acceleration)

Do some specific test to understand what kind of problem. Try run motors without load, does they move or not? Try swap x/y cable, the problem is on y again or switch to x?

**Comment by StuartB4** [comment]
It's harder to engrave on white material because the laser light reflects off it and does not get absorbed in to it as much as darker material. So you need to go slower with more power.

To focus the laser place a black piece of paper on top of the workpiece, put the protective glasses on, switch the laser on low power and adjust the focus ring until you get the smallest dot possible.

**Comment by Bjorn56** [comment]
Ok thanks I will give it a try

________________________________
Van: Stuart <notifications@github.com>
Verzonden: maandag 5 oktober 2020 11:43
Aan: arkypita/LaserGRBL <LaserGRBL@noreply.github.com>
CC: Bjorn56 <bjorn_vosters@hotmail.com>; Author <author@noreply.github.com>
Onderwerp: Re: [arkypita/LaserGRBL] Help wanted Y-axes is not working (#1124)


It's harder to engrave on white material because the laser light reflects off it and does not get absorbed in to it as much as darker material. So you need to go slower with more power.

To focus the laser place a black piece of paper on top of the workpiece, put the protective glasses on, switch the laser on low power and adjust the focus ring until you get the smallest dot possible.
